packages/ui-components/src/components/ListVisual/Rows.js

```diff
@@ -30,7 +30,7 @@ const Row = styled(FlexSpaceBetween)`
   text-align: left;
 
   &:last-child {
-    border-bottom: none;
+    margin-bottom: -1px;
```

Comment:
Shows a border but if there is a border in the element below then this prevents a double border